packages/composites/reference-transform/schemas/env.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/reference-transform-adapter.json",
+  "title": "@chainlink/reference-transform-adapter env var schema",
+  "required": ["SOURCE_DATA_PROVIDER_URL", "RPC_URL"],
+  "type": "object",
+  "properties": {
+    "SOURCE_DATA_PROVIDER_URL": {
```

Comment:
This one is a little weird because it could be any `source` adapter, e.g. `AMBERDATA_DATA_PROVIDER_URL` or `COINGECKO_DATA_PROVIDER_URL`

Reply:
Yeah, i left this as-is for now, as a TODO essentially. Lets open a ticket for this to solve it later on